src/services/sync-service.ts

```diff
@@ -76,13 +77,22 @@ export class SyncService extends EventTarget {
       const syncStartTime = new Date().toISOString();
       
       console.log(lastSyncTimestamp ? `Incremental sync since: ${lastSyncTimestamp}` : 'Full sync');
+      DebugService.log('info', 'sync', 'start', lastSyncTimestamp ? 'Starting incremental sync' : 'Starting full sync', {
```

Comment:
Are the console.log calls still needed (here and elsewhere) now that this is also being logged via the debug service?